api/services/sandbox/v1/sandbox.proto

```diff
@@ -103,6 +104,7 @@ message ControllerCreateRequest {
 	google.protobuf.Any options = 3;
 	string netns_path = 4;
 	map<string, string> annotations = 5;
+	containerd.types.Sandbox sandbox = 6;
```

Comment:
It maybe confuse here because Sandbox has sandbox_id field.
Which one should containerd use for creating?

Reply:
@abel-von @mxpv maybe we can consider to mark containerd.types.Sandbox as object without id. or add some description on CreateRequest and say please ignore the `id` in `sandbox` field.